mailbox/api/src/main/java/org/apache/james/mailbox/model/MailboxAnnotation.java

```diff
@@ -50,6 +64,27 @@ public String getKey() {
         return value;
     }
 
+    private boolean isValidKey(String input) {
```

Comment:
this method checks an input argument, nothing related to the object itself. Usually, it means it's not an object method but a class method.